db/migrate/20171017011510_create_import_record_details.rb

```diff
@@ -0,0 +1,20 @@
+class CreateImportRecordDetails < ActiveRecord::Migration[5.1]
+  def change
+    create_table :import_record_details do |t|
+      t.integer :import_record_id
+      t.string :importer
+      t.datetime :time_started
+      t.datetime :time_ended
+      t.string :status
+      t.string :error_message
+      t.integer :rows_processed
```

Comment:
I like this data structure a lot! @jhilde can you say more about the difference between rows_rejected and rows_excluded? Let's document that as a code comment as well.

Reply:
Ooooh now I see you left an explanation about this in the PR summary -- let's add it as a comment to the codebase too. Probably belongs in `app/models/import_record_detail.rb`.

Reply:
Yup. Sounds like a plan. 

Another part of my importer vision is to make them deal with data inconsistency in a way that doesn't simply abort the import in an indeterminate state.

So yes, to be concrete:
`rows_excluded` right now is the count of rows not in the school filter.
`rows_rejected` is incremented for a row that fails parsing, referential integrity, something like that.